astro/src/content/docs/apis/_application-oauth-configuration-response-body.mdx

```diff
@@ -80,17 +89,66 @@ import JSON from 'src/components/JSON.astro';
 
     The possible values are:
 
-<ApplicationOauthconfigurationProofkeyforcodeexchangepolicy />
+    <ApplicationOauthconfigurationProofkeyforcodeexchangepolicy />
+  </APIField>
+  <APIField name="oauthConfiguration.providedScopePolicy.address.enabled" type="Boolean" since="1.50.0">
+    Whether the `address` OAuth scope provided by FusionAuth is enabled for this application.
```

Comment:
We don't have anything. It's documented elsewhere that FusionAuth doesn't include address in the user data model, but we reserve the scope. There was some debate on whether we should reserve it since we don't have anything to populate, but I thought it was weird to leave that one out.

Reply:
works for me.